src/layouting/text_boundingbox.jl

```diff
@@ -0,0 +1,131 @@
+function boundingbox(plot::Text)
+    @warn """
+    `boundingbox(::Text)` has been deprecated in favor of `Makie.text_boundingbox(::Text)`.
+    In the future `boundingbox(::Text)` will be adjusted to match the other
+    `boundingbox(plot)` functions. The new functionality is currently available
+    as `Makie._boundingbox(plot::Text)`.
+    """
+    return text_boundingbox(plot)
+end
```

Comment:
I'm deprecating the old `boundingbox(::Text)` here so we can make it match the other boundingbox functions in the future.